cli/command/manifest/fetch.go

```diff
@@ -0,0 +1,326 @@
+package manifest
```

Comment:
> unless people go through the process of discovery with godoc or other tools

Exactly, there is actually a well defined way to discover exported things. There is no way to discover unexported code without reading nearly every line of code. This sounds like an argument for exporting things to make them easier to discover.

> Too many Go projects suffer from "over-packagification", including docker. Let's not contribute to it and let's not force it on our contributors.

I would argue most go projects suffer from under-packagification. Without any other language constructs to provide clear boundaries between units, lots of packages are important.